templates/Dataland/v2/ticketing-system-card.html.twig

```diff
@@ -0,0 +1,20 @@
+{% if is_granted('ROLE_DATA_REPOSITORY_MANAGER') %}
+    <div class="py-2">
+        <div class="card text-center text-light">
+            <div class="card-header bg-chart-primary">
+                JIRA Issue
+            </div>
+            <div class="card-body">
+                <p class="card-text text-dark">
+                    {% if dataset.issueTrackingTicket %}
```

Comment:
I don't like the empty block, I think it should have a N/A (or so) placeholder.

Reply:
Yeah, I agree N/A would look better than leaving it empty.  I'll ask William as he's the one that mentioned to leave it empty.